config/default.yaml

```diff
@@ -34,6 +34,12 @@ builds:
     # Upper limit on incoming uploads to builds
     maxByteSize: 1073741824 # 1GB
 
+commands:
+    # How long should commands stay around for
+    expiresInSec: 1814400000 # 3 weeks
```

Comment:
Do we need commands expiration?
A command exists on api side even if the binary of command is expired.

Reply:
Certainly, we don't need commands expiration. However, [catbox-s3 always sets expires parameter](https://github.com/screwdriver-cd/store/blob/master/vendor/catbox-s3/lib/index.js#L236). Therefore, we have to set expiration as long as we use s3-catbox.

Reply:
oh, but I think "3 weeks" is too short.

Reply:
I expand expire time to 1 year.